django/core/checks/registry.py

```diff
@@ -17,6 +17,7 @@ class Tags(object):
     models = 'models'
     security = 'security'
     signals = 'signals'
+    staticfiles = 'staticfiles'
```

Comment:
You could just use the string 'staticfiles' as the tag in the app, there is no restriction to put it on the `Tags` class, this is just intended to be a list of common tags